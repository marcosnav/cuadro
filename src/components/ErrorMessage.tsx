import React, { FC } from 'react';
import * as S from './styled';

const ErrorMessage: FC = () => {
  return (
    <S.Positioner top={78} left={26} right={26}>
      <S.MainTitle>
        Oops!
      </S.MainTitle>
      <S.Text>
        Something went wrong.
        Be sure that you are connected to the internet to be able to load the images used in the puzzles.
      </S.Text>
    </S.Positioner>
  );
};

export default ErrorMessage;
