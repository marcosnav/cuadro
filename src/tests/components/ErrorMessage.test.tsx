import { shallow } from 'enzyme';
import React from 'react';
import ErrorMessage from './../../components/ErrorMessage';

describe('<ErrorMessage/>', () => {
  test('renders properly', () => {
    const component = shallow(<ErrorMessage />);
    expect(component).toMatchSnapshot();
  });
});
