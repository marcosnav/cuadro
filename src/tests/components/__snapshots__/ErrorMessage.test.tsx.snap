// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorMessage/> renders properly 1`] = `
<Positioner
  left={26}
  right={26}
  top={78}
>
  <MainTitle>
    Oops!
  </MainTitle>
  <Text>
    Something went wrong. Be sure that you are connected to the internet to be able to load the images used in the puzzles.
  </Text>
</Positioner>
`;
